CitationStyleLanguagePlugin.inc.php

```diff
@@ -496,33 +550,27 @@ public function loadStyle($styleConfig)
      * software.
      *
      * @param Request $request
-     * @param Submission $article
+     * @param Submission $submission
      * @param string $citationStyle Name of the citation style to use.
      * @param Issue $issue Optional. Will fetch from db if not passed.
      *
      * @return string
      */
-    public function downloadCitation($request, $article, $citationStyle = 'ris', $issue = null)
+    public function downloadCitation($request, $submission, $citationStyle = 'ris', $issue)
```

Comment:
Did you intend to change `$issue = null` to `$issue` in the function signature?

Reply:
No, so we restored it.